data/core/commandview.lua

```diff
@@ -154,6 +154,7 @@ function CommandView:move_suggestion_idx(dir)
     local n = self.suggestion_idx + dir
     self.suggestion_idx = overflow_suggestion_idx(n, #self.suggestions)
     self:complete()
+    self.save_suggestion = self:get_text()
```

Comment:
I guess we should use a different variable, as I think the original purpose of this one was to keep the user inserted text, right?

Reply:
Yeah, I actually don't need the variable, we can just check it against the `suggestions` list.

Reply:
Would that mean that if the user writes the exact text of the item, `suggest` would be called with `""`?

Reply:
Yeah. I think a better way is to just check if the last change was suggestion. Seems to do good.